src/api/user/session.md

```diff
@@ -391,15 +391,15 @@ curl -X DELETE \
   -H "Authorization: token" \
   -H "Content-Type: application/json" \
   -d "{\"sessions\":[\"sid1\",\"sid2\"]}" \
-  https://alekeagle.me/api/sessions
+  https://alekeagle.me/api/user/sessions
```

Comment:
I do appreciate having a second pair of eyes look through my terrible documentation. Thanks WaviestBalloon